ansible/roles/ocp-workload-istio-prometheus-demo/tasks/remove_workload.yml

```diff
@@ -31,7 +31,14 @@
 #    dest: /tmp/istio_product_operator_template.yaml
 
 - name: Remove Istio operator
-  shell: "oc process -f https://raw.githubusercontent.com/Maistra/openshift-ansible/{{istio_maistra_branch}}/istio/istio_{{istio_community_or_product_images}}_operator_template.yaml | oc delete -f -"
+  shell: >-
+    oc process
```

Comment:
This one needs to be cleaned up for readability.

Reply:
This was the format that was suggested to me before for long urls and has been done all over the place. This is a legible way of breaking up long urls with variables in them, since it clearly separates the variable name from the actual url.